services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_scheduler.py

```diff
@@ -356,6 +360,26 @@ async def mark_service_for_removal(
 
         logger.debug("Service '%s' marked for removal from scheduler", service_name)
 
+    async def mark_all_services_in_wallet_for_removal(
+        self, wallet_id: WalletID
+    ) -> None:
+        # TODO: have a test that runs this multiple times in a row, should not do anything bad
```

Comment:
forgotten todo?

Reply:
I've added the test and fixed some flakiness in the module.